documentation/tutorial.md

```diff
@@ -34,14 +43,15 @@ You can list all LXD containers by entering the command `lxc list` in to the com
 +------+-------+------+------+------+-----------+
 ```
 
-## Install and prepare Juju:
+
+## Install and prepare Juju
 [Juju](https://juju.is/) is an operation Lifecycle manager(OLM) for clouds, bare metal, LXD or Kubernetes. We will be using it to deploy and manage Charmed MongoDB. As with LXD, Juju is installed from a snap package:
-```
+```shell
 sudo snap install juju --classic
 ```
 
-Juju already has a built-in knowledge of LXD and how it works, so there is no additional setup or configuration needed. A controller will be used to deploy and control Charmed MongoDB. All we need to do is run the command to bootstrap a Juju controller named ‘overlord’ to LXD. 
-```
+Juju already has a built-in knowledge of LXD and how it works, so there is no additional setup or configuration needed. A controller will be used to deploy and control Charmed MongoDB. All we need to do is run the command to bootstrap a Juju controller named ‘overlord’ to LXD. This bootstrapping processes can take several minutes depending on how provisioned your machine is:
```

Comment:
What does "provisioned" mean in this context?

Reply:
in this case we meant RAM, CPU, etc. Does this change make this more clear?

Reply:
Yes it does, thanks!